config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   root to: "pages#home"
```

Comment:
On s'était dit que notre landing page était /places, il ne faudrait pas le mettre directement ?
Du style `root to: "places#home"` ?
Ou sinon on dégage la root si on en veux  pas ?

Reply:
ok done with root to: "places#index"